Code/Engine/RendererVulkan/CommandEncoder/CommandEncoderImplVulkan.h

```diff
@@ -39,6 +39,8 @@ class EZ_RENDERERVULKAN_DLL ezGALCommandEncoderImplVulkan : public ezGALCommandE
   virtual void InsertFencePlatform(const ezGALFence* pFence) override;
```

Comment:
Note that we would like to remove these fence functions from the high level interface altogether, so don't bother trying to implement them. If you like you can remove them.

Reply:
Will remove them in the next batch of changes.